trip_app/accounts/tests/test_views.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from core.utils import raise_for_status
+from django.urls import reverse
+
+from .user_factory import UserFactory, faker
+
+
+@pytest.fixture
+def signup_data():
+    data = {
+        'username': faker.user_name(),
+        'password': faker.password(),
+        'first_name': faker.first_name(),
+        'last_name': faker.last_name(),
+        'email': faker.email(),
+        'is_staff': False,
+        'is_active': False
+    }
+    return data
+
+
+@pytest.fixture
+def create_user_dict(signup_data, client):
+    """" Create a new user via POST /api/accounts/.
+         Exception will be raised in case of bas response """
+    url = reverse("accounts:create")
+    resp = client.post(url, signup_data)
+    raise_for_status(resp)
+    print(f"user created {resp.json()}")
+    return signup_data
+
+
+@pytest.mark.django_db
+class TestUsers:
+    def test_user_create(self, client, signup_data):
+        """ Verify user is created via GET api/accounts/ """
+        url = reverse("accounts:create")
+        resp = client.post(url, signup_data)
+        assert resp.status_code == 201
+        user_dict = resp.json()
+        print(user_dict)
+        assert user_dict.get('username') == signup_data['username']
+        assert user_dict.get('first_name') == signup_data['first_name']
+        assert user_dict.get('last_name') == signup_data['last_name']
+        assert user_dict.get('email') == signup_data['email']
+
+    @pytest.mark.parametrize('user_qty', [0, 1, 10, 100])
+    def test_user_list(self, admin_client, user_qty):
+        """ Verify that users are displayed with GET api/accounts/ """
+        UserFactory.create_batch(size=user_qty, is_staff=False, )
```

Comment:
this can go to another fixture depending on the `user_qty` fixture. Typically you don't need to create test data within the test case method body.